### ISSUE TITLE
SD card mount location issue

### ISSUE THREAD

**Issue by Syltaris** [ebenbruyns/junkstore#34]
https://github.com/ebenbruyns/junkstore/blob/0b81f5dd810807c25127d46dd10ebd42030a1613/defaults/scripts/Extensions/Epic/settings.sh#L31

^ seems like this path doesn't work for generic SD cards, the plugin will return 'not found' error when attempting to install
maybe instead of hardcoding this, is it possible to sweep/scan the location for potential SD card directories and use that to install instead? 
or maybe provide a way to specify individual SD card locations?

**Comment by ebenbruyns** [comment]
Yeah this one became obvious only after it made contact with a lot of users. I will likely add another option in the settings menu to specify a path. I just need to wrap my head around all the uses cases that are popping up to try and cover as much ground as possible on this one.

**Comment by ebenbruyns** [comment]
You could copy that logic and add it to the overrides file (you can see the path in that same file), that way you can have your own custom solution that won't get clobbered by updates. This is a new concept I added recently so it might change, but it's a way to add "extensions" to extensions.

It's going to be turtles all the way down (if you know the reference).

**Comment by mrsjunkrunner** [comment]
duplicate of #37
